<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>788</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Logged events</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Layer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="layerCombo">
       <item>
        <property name="text">
         <string>Any</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Feature ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="idEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QgsCollapsibleGroupBoxBasic" name="advancedGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Advanced Search</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="insertsChck">
          <property name="text">
           <string>Inserts</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="updatesChck">
          <property name="text">
           <string>Updates</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="deletesChck">
          <property name="text">
           <string>Deletes</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="dataChck">
          <property name="text">
           <string>Data contains</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="dataEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCheckBox" name="afterChck">
          <property name="text">
           <string>After</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="afterDt">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="displayFormat">
           <string>dd/MM/yyyy HH:mm</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="beforeChck">
          <property name="text">
           <string>Before</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="beforeDt">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="displayFormat">
           <string>dd/MM/yyyy HH:mm</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="hsplitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableView" name="eventTable">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QCheckBox" name="onMainCanvas">
           <property name="text">
            <string>Display geometries on the main canvas</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="replayButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Replay</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QSplitter" name="vsplitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTableWidget" name="dataTable">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Nouvelle colonne</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Nouvelle colonne</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Nouvelle colonne</string>
        </property>
       </column>
      </widget>
      <widget class="QgsCollapsibleGroupBoxBasic" name="geometryGroup">
       <property name="title">
        <string>Geometry</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBoxBasic</class>
   <extends>QGroupBox</extends>
   <header>qgis.gui.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <connections>
  <connection>
   <sender>dataChck</sender>
   <signal>toggled(bool)</signal>
   <receiver>dataEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>596</x>
     <y>398</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>afterChck</sender>
   <signal>toggled(bool)</signal>
   <receiver>afterDt</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>433</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>beforeChck</sender>
   <signal>toggled(bool)</signal>
   <receiver>beforeDt</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>433</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
